src/main/java/frc/robot/maps/Shrimp.java

```diff
@@ -0,0 +1,171 @@
+        // Front Left Module
+        final CANcoder encoderFL = new CANcoder(2);
+        CANcoderConfiguration encoderFLConfig = new CANcoderConfiguration();
+        encoderFLConfig.MagnetSensor.AbsoluteSensorRange = AbsoluteSensorRangeValue.Unsigned_0To1;
+        encoderFLConfig.MagnetSensor.MagnetOffset = FLOFFSET;
+        encoderFL.getConfigurator().apply(encoderFLConfig);
+        final SDSSwerveModule frontLeft = new SDSSwerveModule(new Translation2d(MODULE_OFFSET_XY, MODULE_OFFSET_XY),
+                new CtreEncoder(encoderFL), frontLeftSteer, frontLeftDrive, MK4i_L2);
+
+        // Front Right Module
+        final CANcoder encoderFR = new CANcoder(4);
+        CANcoderConfiguration encoderFRConfig = new CANcoderConfiguration();
+        encoderFRConfig.MagnetSensor.AbsoluteSensorRange = AbsoluteSensorRangeValue.Unsigned_0To1;
+        encoderFRConfig.MagnetSensor.MagnetOffset = FROFFSET;
+        encoderFR.getConfigurator().apply(encoderFRConfig);
+        final SDSSwerveModule frontRight = new SDSSwerveModule(new Translation2d(MODULE_OFFSET_XY, -MODULE_OFFSET_XY),
+                new CtreEncoder(encoderFR), frontRightSteer, frontRightDrive, MK4i_L2);
+
+        // Rear Left Module
+        final CANcoder encoderRL = new CANcoder(1);
+        CANcoderConfiguration encoderRLConfig = new CANcoderConfiguration();
+        encoderRLConfig.MagnetSensor.AbsoluteSensorRange = AbsoluteSensorRangeValue.Unsigned_0To1;
+        encoderRLConfig.MagnetSensor.MagnetOffset = RLOFFSET;
+        encoderRL.getConfigurator().apply(encoderRLConfig);
+        final SDSSwerveModule rearLeft = new SDSSwerveModule(new Translation2d(-MODULE_OFFSET_XY, MODULE_OFFSET_XY),
+                new CtreEncoder(encoderRL), rearLeftSteer, rearLeftDrive, MK4i_L2);
+
+        // Rear Right Module
+        final CANcoder encoderRR = new CANcoder(3);
+        CANcoderConfiguration encoderRRConfig = new CANcoderConfiguration();
+        encoderRRConfig.MagnetSensor.AbsoluteSensorRange = AbsoluteSensorRangeValue.Unsigned_0To1;
+        encoderRRConfig.MagnetSensor.MagnetOffset = RROFFSET;
+        encoderRR.getConfigurator().apply(encoderRRConfig);
+        final SDSSwerveModule rearRight = new SDSSwerveModule(new Translation2d(-MODULE_OFFSET_XY, -MODULE_OFFSET_XY),
+                new CtreEncoder(encoderRR), rearRightSteer, rearRightDrive, MK4i_L2);
+
+        final double maxDriveSpeedMetersPerSecond = Units.feetToMeters(14.5);
+
+        final double maxRotationRadianPerSecond = Math.PI * 2;
+
+        final HolonomicPathFollowerConfig config = new HolonomicPathFollowerConfig(
+                // HolonomicPathFollowerConfig, this should likely live in your
+                // Constants class
+                new PIDConstants(0, 0, 0), // Translation PID constants (OFF_AXIS)
+                new PIDConstants(0, 0.0, 0.0), // Rotation PID constants (OFF_AXIS)
+                2.0, // Max module speed, in m/s
+                0.2155317,
```

Comment:
Would be nice to update this to calculate the hypotenuse from `MODULE_OFFSET_XY`